packages/eslint-plugin-react-hooks/__tests__/ESLintRulesOfHooks-test.js

```diff
@@ -913,12 +913,21 @@ function loopError(hook) {
 }
 
 function functionError(hook, fn) {
-  return {
-    message:
-      `React Hook "${hook}" is called in function "${fn}" that is neither ` +
-      'a React function component nor a custom React Hook function.' +
-      ' React component names must start with an uppercase letter.',
-  };
+  if (hook[0] === hook[0].toLowerCase()) {
```

Comment:
Since this is a helper function, we should split it out into a different function so we have both:

- `functionError`
- `functionInvalidError`

That way we can assert exactly which one we expect.

Reply:
Ah, thank you, I have just been experimenting with getting a build of react with my changes working, I will change that.